hwdb.d/60-evdev.hwdb

```diff
@@ -732,3 +732,13 @@ evdev:input:b0003v172Fp0047*
 evdev:input:b0003v172Fp0505*
  EVDEV_ABS_00=::160
  EVDEV_ABS_01=::160
+ 
+#########################################
+# Positivo-Vaio
+#########################################
+# Vaio FE14
+evdev:name:SYNA3602:00 0911:5288 Touchpad:dmi:bvn*:bvr*:bd*:svnPositivoBahia-VAIO:pnVJFE41F11*:pvr*
```

Comment:
```suggestion
evdev:name:SYNA3602:00 0911:5288 Touchpad:dmi:*svnPositivoBahia-VAIO:pnVJFE41F11*
```
No need to specify all the others with globs.

Reply:
hwdb files are all parsed by the same parser, if that's what you're referring to. Whether you specify all sub-components is just convention, looks like `60-keyboard.hwdb` primarily uses the long forma, `60-evdev.hwbd` primarily uses the shortened format. I think it's better to use the short version here for consistency (plus: it's less visual detritus).

Reply:
thanks!